stdlib/2/typing.pyi

```diff
@@ -385,7 +385,8 @@ class Pattern(Generic[AnyStr]):
 
 # Functions
 
-def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+def get_type_hints(obj: Callable, globalns: Optional[dict[AnyStr, Any]],
```

Comment:
I don't think AnyStr is right; there's nothing stopping users from using a mix of str and unicode in Python 2, and using bytes in Python 3 almost certainly won't work. That's why I suggested Text before. Also, come to think of it the argument types should probably be Mapping, not just dict.